slurm/merge.jl

```diff
@@ -30,17 +33,99 @@ function main(config::Dict)
         # Save dataset to disk
         get!(D, "meta", Dict{String,Any}())
         D["meta"]["config"] = config_str
-        OPFGenerator.save_h5(joinpath(export_dir, "$(casename)_$(dataset_name).h5"), D);
+
+        if dataset_name == "input"
+            OPFGenerator.save_h5(joinpath(export_dir, "input.h5"), D)
+        else
+            mkpath(joinpath(export_dir, dataset_name))
+            OPFGenerator.save_h5(joinpath(export_dir, dataset_name, "meta.h5"), D["meta"])
+            OPFGenerator.save_h5(joinpath(export_dir, dataset_name, "primal.h5"), D["primal"])
+            OPFGenerator.save_h5(joinpath(export_dir, dataset_name, "dual.h5"), D["dual"])
+        end
 
         GC.gc()
     end
 
-    return nothing    
+    rm(joinpath(export_dir, "res_h5"), recursive=true)
+
+    rng = MersenneTwister(42)
+
+    feasible = nothing
+    n_samples = nothing
+
+    # find inputs that are feasible for all OPFs
```

Comment:
It may be worth making this section optional, via something like `julia --project=. slurm/merge.jl --no-split`.